apps/netlify/lambda/serverless.yml

```diff
@@ -0,0 +1,47 @@
+service: sls-apps-netlify-build-action
+
+frameworkVersion: ">=2.0.0 <3.0.0"
+
+plugins:
+  - serverless-domain-manager
+
+custom:
+  signingSecret:
+    prd: ${ssm:/aws/reference/secretsmanager/ci/apps/netlify-build-action/prd/signing-secret}
+    test: ${ssm:/aws/reference/secretsmanager/ci/apps/netlify-build-action/test/signing-secret}
+  appIdentityKey:
+    prd: ${ssm:/aws/reference/secretsmanager/ci/apps/netlify-build-action/prd/app-identity-key}
+    test: ${ssm:/aws/reference/secretsmanager/ci/apps/netlify-build-action/test/app-identity-key}
+  customDomain:
+    domainName: netlify-build-action.ctfapps.net
+    stage: prd
+    createRoute53Record: true
+    endpointType: "edge"
+    securityPolicy: tls_1_2
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: ${opt:stage, 'test'}
+  region: "us-east-1"
+  deploymentBucket:
+    name: cf-apps-serverless-deployment
+  deploymentPrefix: sls-apps-netlify-build-action
+
+package:
+  patterns:
+    - "!node_modules/@contentful/netlify-build-and-preview/node_modules/**"
+
+functions:
+  app:
+    handler: index.handler
+    environment:
+      SIGNING_SECRET: ${self:custom.signingSecret.${opt:stage, 'test'}}
+      APP_IDENTITY_PRIVATE_KEY: ${self:custom.appIdentityKey.${opt:stage, 'test'}}
+    events:
+      - http:
+          path: /
+          method: ANY
```

Comment:
Do we need both the routes? Not sure how serverless works, but this might be spawning two lambdas?